src/main/java/com/example/weather/client/models/entity/Main.java

```diff
@@ -0,0 +1,21 @@
+package com.example.weather.client.models.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Main {
+    private Double temp;
+    @JsonProperty("temp_min")
+    private Double tempMin;
+    @JsonProperty("temp_max")
+    private Double tempMax;
```

Comment:
```suggestion
    private Double tempMin;
    private Double tempMax;
```
This feature is not for handling naming strategy differences. `@JsonProperty` is rather for overwriting odd field names in JSON (like `dt` in `WeatherDao`).